app/lib/atom_serializer.rb

```diff
@@ -16,7 +16,7 @@ def render(element)
   def author(account)
     author = Ox::Element.new('author')
 
-    uri = TagManager.instance.uri_for(account)
+    uri = TagManager.instance.uri_for(account).gsub(Rails.configuration.x.web_domain, Rails.configuration.x.local_domain)
```

Comment:
I agree.
However, I don't know where does mastodon use `'TagManager.instance.uri_for`.
So, I can't guess where will changing `TagManager.instance.uri_for` affect.

Should I move that to `uri_for`?
If `TagManager.instance.uri_for` will be changed, I insert `gsub` in [here](https://github.com/tootsuite/mastodon/blob/v1.2/app/lib/tag_manager.rb#L83-L84).